doc/news/changes.h

```diff
@@ -159,6 +159,12 @@ inconvenience this causes.
 <a name="general"></a>
 <h3>General</h3>
 <ol>
+  <li> New: Triangulation::ghost_neighbors() returns a list of MPI ranks of owners
```

Comment:
I like std::set better. It automatically answers the question of duplicates and is sorted for when you need that.

As for `memcpy`: I prefer `std::copy` anyway :-)